config/routes.rb

```diff
@@ -13,17 +13,16 @@
 
     resources :account_activations, only: :edit
     resources :password_resets, only: %i(new create edit update)
-    resources :users, only: %i(show edit update)
-
-    resources :courses, only: :show do
-      member do
-        get :members
-        get :subjects
-      end
-    end
+    resources :users
```

Comment:
```suggestion
    resources :users, only: %i(show edit update)
```
giữ router của users như cũ, trong controller k thấy có action mới